### ISSUE TITLE
Pre-trained model?

### ISSUE THREAD

**Issue by ricshaw** [ChengBinJin/semantic-image-inpainting#2]
Thanks for the code!
Are you able to provide a pre-trained model for either the celebA or svhn datasets?

**Comment by ChengBinJin** [comment]
You can download the weights of DCGAN for celebA and svhn datasets from the following link.
https://www.dropbox.com/sh/q0bqg9m0ggdp1vv/AACXIfsnFomG5xJHNBxM23UAa?dl=0
I didn't fully tunning the DCGAN, so if you re-train DCGAN many times you can get more good network that applied is image inpainting.

**Comment by ChengBinJin** [comment]
> Thanks for the code!
> Are you able to provide a pre-trained model for either the celebA or svhn datasets?

You can download the weights of DCGAN for celebA and svhn datasets from the following link.
https://www.dropbox.com/sh/q0bqg9m0ggdp1vv/AACXIfsnFomG5xJHNBxM23UAa?dl=0
I didn't fully tunning the DCGAN, so if you re-train DCGAN many times you can get more good network that applied in image inpainting.

**Comment by qivigor** [comment]
Thanks for the code!
can you release some images from your test set? I don't know which image can be used to test your model in celeba dataset.

**Comment by ChengBinJin** [comment]
Images in the celebA dataset are numbered from 000001 to 202599. The images were manually separated into training and validation set. The first 2000 images (from 0000001 to 002000) were saved in the validation set, and the remaining (from 002001 to 202599) were included in the training. The DCGAN model was trained on training data, and inpainting model was evaluated on validation data.

**Comment by qivigor** [comment]
> Images in the celebA dataset are numbered from 000001 to 202599. The images were manually separated into training and validation set. The first 2000 images (from 0000001 to 002000) were saved in the validation set, and the remaining (from 002001 to 202599) were included in the training. The DCGAN model was trained on training data, and inpainting model was evaluated on validation data.

Thanks!!!
